src/components/Incidents/IncidentList/index.js

```diff
@@ -1,29 +1,146 @@
+            <th
+              className="px-6 py-3 text-medium-gray font-medium text-xs col-span-2 text-left"
+              colSpan={2}
+            >
+              Name
+            </th>
+            <th className="px-3 py-3 text-medium-gray font-medium text-xs text-left">
+              Severity
+            </th>
+            <th className="px-3 py-3 text-medium-gray font-medium text-xs text-left">
+              Status
+            </th>
+            <th className="px-3 py-3 text-medium-gray font-medium text-xs text-left">
+              Age
+            </th>
+            <th
+              className="px-3 py-3 text-medium-gray font-medium text-xs col-span-2 text-left"
+              colSpan={2}
+            >
+              Responders
+            </th>
+          </tr>
+        </thead>
+        <tbody className="flex-1 overflow-y-auto">
+          {list.map((incident) => (
+            <IncidentItem incident={incident} key={uuid()} />
+          ))}
+        </tbody>
+      </table>
     </div>
   );
 }
 
-function IncidentItem({ incident, ...rest }) {
-  const { title, id, description, severity, status, type } = incident;
+function IncidentItem({ incident }) {
+  const { title, id, created_at: createdAt, severity, status } = incident;
+  const age = dayjs(createdAt).fromNow();
+  const navigate = useNavigate();
+  const navigateToIncidentDetails = (id) => {
+    navigate(`/incidents/${id}`);
+  };
+
+  const severityInfo = useMemo(() => {
+    switch (severity) {
+      case 0:
+        return severityData.low;
+      case 1:
+        return severityData.high;
+      default:
+        return severityData.low;
+    }
+  }, [severity]);
+
+  const statusLabel = useMemo(
+    () => IncidentStatusLabel[status] ?? status,
+    [status]
+  );
+
+  const statusColorClass = cx({
+    "bg-light-green": status === IncidentStatus.Open,
+    "bg-gray-100": status === IncidentStatus.Closed
+  });
+
   return (
-    <Link to={`/incidents/${id}`} {...rest}>
-      <div className="px-6 py-3">
-        <div className="text-gray-800 font-semibold">{title}</div>
-        <div className="text-gray-400 text-sm">description: {description}</div>
-        <div className="text-gray-400 text-sm">id: {id}</div>
-        <div className="text-gray-400 text-sm">severity: {severity}</div>
-        <div className="text-gray-400 text-sm">status: {status}</div>
-        <div className="text-gray-400 text-sm">type: {type}</div>
-      </div>
-    </Link>
+    <tr
+      className="last:border-b-0 border-b cursor-pointer"
+      onClick={() => navigateToIncidentDetails(id)}
+    >
+      <td
+        colSpan={2}
+        className="px-6 py-4 text-darker-black col-span-2 text-sm leading-5 font-medium"
+      >
+        {title}
+      </td>
+      <td className="px-3 py-3">
+        <div className="flex flex-row items-center">
+          <Icon name={severityInfo.icon} />
```

Comment:
i decided this is more universal. heroicons may not contain all the required icons

Reply:
We use https://react-icons.github.io which has pretty much all free icon sets - The included icons are intended primarily for product logos.

Reply:
@moshloop done. Anything else here ?